### DISCUSSION TITLE
Optional argument in mailer

### DISCUSSION THREAD

**Discussion by vlado** [janko/rodauth-rails#157] [General]
Hi @janko, big thanks for the effort to make it easier for everyone to use Rodauth with Rails.

I'm using Rodauth for the first time and I managed to setup and configure everything without any problems 🙇 but once I've run Rubocop I got one interesting offence

```
app/mailers/rodauth_mailer.rb:4:22: C: Style/OptionalArguments: Optional arguments should appear at the end of the argument list.
  def verify_account(name = nil, account_id, key)
```

I do not always agree with Rubocop but this time it seems reasonable what it is suggesting. From what I see it is not needed for `name` to be optional argument cause name is always passed (via self.class.configuration_name)?

**Comment by janko** [comment]
Thanks for the suggestion. I made it optional for convenience if someone wants to test the mailer manually, but I think most people will test emails when calling Rodauth actions. So removing the optional argument seems resonable 👍
